### PR TITLE
Robotframework lint

### PR DESCRIPTION
**Pull request by raphaelcastaneda**

### CODE REVIEW

File: inlineplz/parsers/rflint.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    def parse(self, lint_data):
+        messages = set()
+        current_file = None
+        for line in lint_data.split('\n'):
+            try:
+                if line.startswith('+'):
+                    current_file = line.split(' ')[1]
+                    continue
+                else:
+                    severity, position, message = line.split(':')
```
Comment (guykisel) [inline inlineplz/parsers/rflint.py]: `prospector: pylint: Unused variable 'severity' (unused-variable)`

Comment (raphaelcastaneda) [inline inlineplz/parsers/rflint.py]: fixed

File: requirements_dev.txt [hunk 2]
```diff
@@ -10,4 +10,4 @@ Sphinx==1.3.1
 PyYAML==3.11
 pytest==2.8.7
 prospector==0.11.7
-
+robotframework-lint==0.7
```
Comment (guykisel) [inline requirements_dev.txt]: i havent generally been putting the linters in the dev requirements but maybe thats not a bad idea

File: inlineplz/linters/__init__.py [hunk 3]
```diff
@@ -154,6 +155,17 @@
         'autorun': True,
         'run_per_file': False
     },
+    'rflint': {
+        'install': [['pip', 'install', 'robotframework-lint']],
+        'help': ['rflint', '--help'],
+        'run': ['rflint', '.'],
+        'rundefault': ['rflint', '-A', '{config_dir}/.rflint', '.'],
+        'dotfiles': [],
```
Comment (guykisel) [inline inlineplz/linters/__init__.py]: might want to put `.rflint` here

**Pull request merged**